src/disc/stk/Albany_ExtrudedSTKMeshStruct.cpp

```diff
@@ -327,7 +327,7 @@ void Albany::ExtrudedSTKMeshStruct::setBulkData(
   Teuchos::RCP<const Tpetra_Map> sides_map = Tpetra::createNonContigMapWithNode<LO, Tpetra_GO, KokkosNode>(indices(),comm);
 
   GO globalElemStride = cells_map->getMaxAllGlobalIndex() - cells_map->getMinAllGlobalIndex() + 1;
-  GO globalVerticesStride = nodes_map->getMaxAllGlobalIndex() - nodes_map->getMinAllGlobalIndex() + 1;
+  GO globalVerticesStride = nodes_map->getMaxAllGlobalIndex() + 1;
```

Comment:
I think that the definition of the globalVerticesStride is correct. Maybe it's not used as it should.